src/globals/helpers/_media-queries.scss

```diff
@@ -11,6 +11,6 @@ $sass-mq-already-included: false !default;
   $mq-show-breakpoints: ();
 }
 
-@import "./node_modules/sass-mq/mq";
+@import "sass-mq/mq";
```

Comment:
the webpack sass-loader way of doing this would be `"~sass-mq/mq"` https://github.com/webpack-contrib/sass-loader#imports

Reply:
Is there any reason not to use the `~` syntax here? I think the current implementation would mean that the parent project would need to manually ensure `sass-mq` was on the resolution path for sass files, wheras `~` should always work as long as the parent project used sass loader.

Or are you saying that compatibility with Sass build systems that don't support `~` is more important than manual setup?

Would be good to see how existing Sass libraries distributed via npm that have Sass sub-dependencies handle this.

Reply:
Do you know what the support is for that syntax? We can make very few assumptions about the technologies involved our consumer's build pipelines, so unless that syntax is very well supported I would be wary of relying on it.

Reply:
I think the problem (as described at https://github.com/webpack-contrib/sass-loader#imports) is Sass doesn't have a default syntax for library imports.

The 3 options are:
1. `~`
2.`node_modules` or `./node_modules`
3.  mixing relative + lib paths (`@import "sass-mq/mq";`)

I think all 3 either infer an assumption or requirement from the parent project.

I'll take a look around and see if I can find examples in other Sass projects dealing with this.

Reply:
related, but not found an answer yet:

https://github.com/sass/node-sass/issues/1596
https://github.com/webpack-contrib/sass-loader/issues/466

Reply:
~ is webpack specific, node-sass doesnt supoort that

Reply:
I made a couple comments on sass/node-sass#1596 and also found and commented on sass/sass/issues#2350

Is it ok to assume the parent project will use `node-sass` (given that it's distributed via npm)?

Looks like [eyeglass](https://github.com/sass-eyeglass/eyeglass) may be the recommended solution if so.